cdk_framework/EKS/lib/utils/validate.ts

```diff
@@ -0,0 +1,147 @@
+const yaml = require('js-yaml')
+
+const supportedFields = new Set(['version', 'ec2_instance', 'launch_type', 'node_size'])
+const supportedVersions = new Set(['1.18', '1.19', '1.20', '1.21']);
```

Comment:
```suggestion
const supportedVersions = new Set(['1.18', '1.19', '1.20', '1.21']);
```
Are these the actual supported Kubernetes versions? Or do they have patch release versions that can be targeted?